nvi-commons/src/main/java/no/sikt/nva/nvi/common/db/CandidateRepository.java

```diff
@@ -61,7 +61,7 @@ public CandidateDao createDao(DbCandidate dbCandidate, List<DbApprovalStatus> ap
         return candidateTable.getItem(candidate);
     }
 
-    public void updateCandidate(UUID identifier, CandidateDao candidateDao, List<DbApprovalStatus> approvals) {
+    public void updateCandidateResettingApprovals(UUID identifier, CandidateDao candidateDao, List<DbApprovalStatus> approvals) {
```

Comment:
Not sure I agree with this renaming. The repository should not contain business logic. This method takes in `identifier`, `candidateDao` and `approvals`, and updates rows in the database according to this. It does not contain "resetting"-logic, it just updates the approvals in the database.
You could rename it to `updateCandidateAndApprovals` if you want to make it clear that it updates approvals as well?

Reply:
User who takes this methods in use wants to know what these methods do, calling all methods update() and passing in different variables makes things more complicated. Method name should say what it actually does. This method updates candidate and sets approvals to its default value.